ecs/daemon/parsers.conf

```diff
@@ -0,0 +1,23 @@
+[PARSER]
```

Comment:
TODO: I think we can just use the built-in docker parser. Only syslog may be needed here. Testing this.